modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -0,0 +1,138 @@
+$ kubectl apply -f memcachedoperator.0.0.1.csv.yaml
+$ kubectl get ClusterServiceVersion-v1s memcachedoperator.v0.0.1 -o json | jq '.status'
+----
+
+.. After applying this manifest, nothing has happened yet, because the cluster does
+not meet the requirements specified in our manifest. Create the
+`CustomResourceDefinition` and RBAC rules for the Memcached type managed by the
+Operator:
++
+----
+$ kubectl apply -f deploy/rbac.yaml
+$ kubectl apply -f deploy/operator.yaml
+----
++
+Because the OLM creates Operators in a particular namespace when a manifest is
+applied, administrators can leverage the native Kubernetes RBAC permission model
+to restrict which users are allowed to install Operators.
+
+. *Create an application instance.*
++
+The Memcached Operator is now running in the `memcached` namespace. Users
+interact with Operators via instances of `CustomResources`; in this case, the
+resource has the kind `Memcached`. Native Kubernetes RBAC also applies to
+`CustomResources`, providing administrators control over who can interact with
+each Operator.
++
+Creating instances of Memcached in this namespace will now trigger the Memcached
+Operator to instantiate pods running the memcached server that are managed by
+the Operator. The more `CustomResources` you create, the more unique instances
+of Memcached are managed by the Memcached Operator running in this namespace.
++
+----
+$ cat <<EOF | kubectl apply -f -
+apiVersion: "cache.example.com/v1alpha1"
+kind: "Memcached"
+metadata:
+  name: "memcached-for-wordpress"
+spec:
+  size: 1
+EOF
+
+$ cat <<EOF | kubectl apply -f -
+apiVersion: "cache.example.com/v1alpha1"
+kind: "Memcached"
+metadata:
+  name: "memcached-for-drupal"
+spec:
+  size: 1
+EOF
+
+$ kubectl get Memcached
+NAME                      AGE
+memcached-for-drupal      22s
+memcached-for-wordpress   27s
+
+$ kubectl get pods
+NAME                                       READY     STATUS    RESTARTS   AGE
+memcached-app-operator-66b5777b79-pnsfj    1/1       Running   0          14m
+memcached-for-drupal-5476487c46-qbd66      1/1       Running   0          3s
+memcached-for-wordpress-65b75fd8c9-7b9x7   1/1       Running   0          8s
+----
+
+. *Update an application.*
++
+Manually applying an update to the Operator is as simple as creating a new
+Operator manifest with a `replaces` field that references the old Operator
+manifest. The OLM ensures that all resources being managed by the old Operator
+have their ownership moved to the new Operator without fear of any programs
+stopping execution. It is up to the Operators themselves to execute any data
+migrations required to upgrade resources to run under a new version of the
+Operator.
++
+The following command demonstrates applying a new
+link:https://github.com/operator-framework/getting-started/blob/master/memcachedoperator.0.0.2.csv.yaml[Operator manifest file]
+using a new version of the Operator and shows that the pods remain executing:
++
+----
```

Comment:
We also need to update this file: https://github.com/operator-framework/getting-started/blob/master/memcachedoperator.0.0.2.csv.yaml, the same reason as the above shows.

Reply:
Addressed via https://github.com/operator-framework/getting-started/pull/22.